pipeline/extract.py

```diff
@@ -8,33 +9,62 @@
 TIME = "time"
 
 
-def get_minute_from_epoch_time(time_in_ms: int) -> datetime:
+def get_time_from_epoch_time(time_in_ms: int) -> datetime:
```

Comment:
```suggestion
def get_time_from_epoch_time(time_in_ms: int) -> str:
```

I think this function returns a string instead of a datetime object